dojo/templates/dojo/findings_list_snippet.html

```diff
@@ -312,10 +312,22 @@ <h3 class="has-filters">
                                     <th class="nowrap centered severity-sort">
                                         {% trans "Severity" %}
                                     </th>
+                                    {% if show_product_column and product_tab is None %}
```

Comment:
New order is:

- severity
- product
- vuln name
- security scanner (found by)
- status (active, verified, etc)